node/src/accountant/scanners.rs

```diff
@@ -1946,26 +2436,29 @@ mod tests {
         financial_statistics.total_paid_payable_wei += 1111;
         subject.financial_statistics.replace(financial_statistics);
 
-        subject.confirm_transaction(fingerprint.clone(), &Logger::new(test_name));
+        subject.confirm_transactions(
+            vec![fingerprint_1.clone(), fingerprint_2.clone()],
+            &Logger::new(test_name),
+        );
 
         let total_paid_payable = subject.financial_statistics.borrow().total_paid_payable_wei;
-        let transaction_confirmed_params = transaction_confirmed_params_arc.lock().unwrap();
-        assert_eq!(total_paid_payable, 1111 + 5478);
-        assert_eq!(*transaction_confirmed_params, vec![fingerprint])
+        let transaction_confirmed_params = transactions_confirmed_params_arc.lock().unwrap();
+        assert_eq!(total_paid_payable, 1111 + 5478 + 6543);
+        assert_eq!(
```

Comment:
I get the feeling that this was already asserted in another test. If so, it doesn't specifically hurt anything to assert it here too, except that if the code changes, now you'll need to change the tests in more places than strictly necessary.